scraper.rb

```diff
@@ -6,18 +6,12 @@
 require 'nokogiri'
 require 'open-uri'
```

Comment:
You've left behind quite a few unnecessary 'require's here — e.g. you removed the function that uses 'date', but not removed the `require date`